Sources/idt/idt.cc

```diff
@@ -105,19 +200,21 @@ class visitor : public clang::RecursiveASTVisitor<visitor> {
   template <typename Decl_>
   bool is_in_header(const Decl_ *D) const {
     const clang::FullSourceLoc location = get_location(D);
-  const clang::FileID id = source_manager_.getFileID(location);
-  if (const auto entry = source_manager_.getFileEntryRefForID(id)) {
-    const llvm::StringRef name = entry->getName();
-    for (const auto &extension : { ".h", ".hh", ".hpp", ".hxx" })
-      if (name.ends_with(extension))
-        return true;
+    const clang::FileID id = source_manager_.getFileID(location);
+    if (const auto entry = source_manager_.getFileEntryRefForID(id)) {
+      const llvm::StringRef name = entry->getName();
+      for (const auto &extension : {".h", ".hh", ".hpp", ".hxx"})
+        if (name.ends_with(extension))
+          return true;
     }
     return false;
   }
 
 public:
-  explicit visitor(clang::ASTContext &context)
-      : context_(context), source_manager_(context.getSourceManager()) {}
+  explicit visitor(clang::ASTContext &context,
```

Comment:
We can drop the `explicit` here - it is no longer a single parameter constructor.